src/Neo.VM/Neo.VM.csproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Copyright>2016-2023 The Neo Project</Copyright>
+    <Description>Neo virtual machine</Description>
+    <VersionPrefix>3.6.2</VersionPrefix>
+    <Authors>The Neo Project</Authors>
+    <TargetFrameworks>netstandard2.1;net7.0</TargetFrameworks>
+    <LangVersion>9.0</LangVersion>
+    <PackageTags>NEO;AntShares;Blockchain;Smart Contract;VM</PackageTags>
+    <PackageProjectUrl>https://github.com/neo-project/neo-vm</PackageProjectUrl>
+    <PackageLicenseExpression>MIT</PackageLicenseExpression>
```

Comment:
Likely some other metadata can be shared with the main Neo package, like Neo.Json does.